src/locale/zh-CN.h

```diff
@@ -453,7 +453,7 @@
 #endif  // D_STR_MESGDESC
 #ifndef D_STR_IRRECVDUMP_STARTUP
 #define D_STR_IRRECVDUMP_STARTUP \
-    "IRrecvDumpV2 运行当中，等待红外信息输入位于引脚 %d"
+    "IRrecvDump 运行当中，等待红外信息输入位于引脚 %d"
```

Comment:
Should The // IRrecvDumpV2 comment above also be changed? (in this file at line 444)

Reply:
Done.